### ISSUE TITLE
Change primitive should be more robust.

### ISSUE THREAD

**Issue by indexzero** [nodejitsu/godot#4]
Right now something like:

``` js
  var godot = require('godot');

  godot.createServer({
    reactors: [
      godot.reactor()
        .change('state')
    ]
  });
```

Can only listen when the `state` property on data _changes_, but not when it changes **from one specific value to another**

**Comment by jcrugzz** [comment]
So for this particular example you are talking about if `state` changed from `warning` to `ok` or `warning` to `critical`?

**Comment by indexzero** [comment]
right. I would expect something like:

``` js
  var godot = require('godot');

  godot.createServer({
    reactors: [
      godot.reactor()
        .change('state', { from: 'ok', to: 'critical' })
    ]
  });
```

**Comment by jcrugzz** [comment]
Ok that makes sense. What about for the case of multiple keys? Something like

``` js
 var godot = require('godot');

  godot.createServer({
    reactors: [
      godot.reactor()
        .change({key: 'state', from: 'ok', to: 'critical' }, {key: 'description'} )
    ]
  });
```

would work but the api is not as clean. Any suggestions?

[edit] Thinking about it more this wouldn't really be a practical use case since you would probably create a separate reactor. So I guess this may go into if you want to keep the current multi key aspect of the change reactor. All depends on how its being used and `state` is definitely an ideal use case.

**Comment by jcrugzz** [comment]
fixed in #31
